### PR TITLE
refactor: run provision-react-component

### PR DESCRIPTION
**Pull request by keithamus**

### CODE REVIEW

**Comment by gcedo** [comment]
Can you please update 

```
@economist/component-icon
@economist/component-balloon
@economist/component-google-search
```

to the latest version?

File: src/example.js [hunk 1]
```diff
@@ -1,14 +1,13 @@
-if (typeof document === 'object') {
-  require('svg4everybody')();
-}
-
-import Navigation from './index';
+import Navigation from './';
 import React from 'react';
-// Get data.
-import context from '@economist/component-sections-card/context';
+import navgiationLinks from '@economist/component-sections-card/context';
```
Comment (umbobabo) [inline src/example.js]: typo

File: src/index.js [hunk 2]
```diff
@@ -1,61 +1,72 @@
...
+        href: React.PropTypes.string,
+      }),
+    }).isRequired,
+    sectionsCardData: SectionsCard.propTypes.data,
+    accordionData: Accordion.propTypes.list,
   }
 
   static defaultProps = {
     autohide: true,
     penName: 'guest-olejses',
```
Comment (umbobabo) [inline src/index.js]: Does make sense have a default penName?

Comment (keithamus) [inline src/index.js]: Probably not. I'll remove

**Comment by umbobabo** [comment]
After removed the node_modules folder the install process end up with an error:

> ghooks@1.0.3 install /Users/imac/e-comp/component-navigation/node_modules/ghooks
> node ./bin/install

fs.js:585
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open '/Users/imac/e-comp/component-navigation/.git/hooks/pre-commit'
    at Error (native)
    at Object.fs.openSync (fs.js:585:18)
    at Object.fs.writeFileSync (fs.js:1225:33)
    at install (/Users/imac/e-comp/component-navigation/node_modules/ghooks/lib/install.js:57:6)
    at Array.forEach (native)
    at installHooks (/Users/imac/e-comp/component-navigation/node_modules/ghooks/lib/install.js:42:11)
    at Object.<anonymous> (/Users/imac/e-comp/component-navigation/node_modules/ghooks/bin/install:3:1)
    at Module._compile (module.js:399:26)
    at Object.Module._extensions..js (module.js:406:10)
    at Module.load (module.js:345:32)

**Comment by keithamus** [comment]
Try running this:

```bash
rm -rf .git/hooks/*
cd node node_modules/ghooks && node lib/install.js
cd - 
```

**Comment by gcedo** [comment]
`npm i` went smooth, all tests pass. :rainbow:

**Pull request merged**